compose/service.py

```diff
@@ -249,6 +249,19 @@ def create_container(self,
 
         return Container.create(self.client, **container_options)
 
+    def exec_create(self, command, **kwargs):
+        try:
+            container = self.get_container(number=kwargs["index"])
+        except KeyError:
+            container = self.get_container()
+        return Container.create_exec(self.client, container.id, command, **kwargs)
+
+    def exec_start(self, exec_id, **kwargs):
+        return Container.start_exec(self.client, exec_id, **kwargs)
+
+    def get_exec_exit_code(self, exec_id):
+        return self.client.exec_inspect(exec_id).get("ExitCode")
```

Comment:
Since these are container operations I think we should omit these 3 methods and just call the `Container` methods directly from the cli command. The `port` command is an example of this. It grabs the correct container first then operates directly on the container instead of the service.